### DISCUSSION TITLE
Is this used in Production systems currently?

### DISCUSSION THREAD

**Discussion by elliotweaver** [wesone/blackrik#2] [Q&A]
We are evaluating this project currently and it seems great! I am wondering if, in its current state, it is production ready. Has it been successfully used in a production system and if so how has it been performing?

**Comment by wesone** [comment]
As of now there is no production system running yet. All projects using Blackrik are currently in development, however all current features are used and work properly there but I can't guarantee that it is bug-free.

Feel free to suggest new features!
